addons/mail/static/src/core_ui/message.js

```diff
@@ -150,6 +153,8 @@ export class Message extends Component {
                 $(this.messageBody.el).find(".o-mail-read-more-less").remove();
             }
         });
+
+        this.messageReactionViewer = this.messageReactionViewer();
```

Comment:
Just `reactionViewer` is good (`message` prefix is redundant with name of component `Message`)